### ISSUE TITLE
Slide menu visual bug pushing and popping view controllers

### ISSUE THREAD

**Issue by nahuelDeveloper** [aryaxt/iOS-Slide-Menu#251]
I updated my project to Swift 4 to support iOS 11 in my application and im experiencing a strange behaviour when pushing and popping view controllers created programmatically. (In my project I don't use Storyboards or Nibs, so all my view controllers are created programmatically)

When I push a view controller created programmatically onto the SlideNavigationController's stack, the transition animation looks strange. The same thing happens when it is popped from the stack.

I have another pod, a loader (CCActivityHUD), that works fine until I push a programmatically created view controller for the first time. After that, it stops showing on any screen.

I tried using a view controller created via Storyboard and using the method 'instantiateViewControllerWithIdentifier' to access it, and the visual bug disappears!

Has anybody experienced a similar behaviour?

**Comment by juliusbtesh** [comment]
Yes, I am also getting a weird behavior that I did not have before the update.
I am using all programmatically created view controllers, no XIBs
